common/cond_handle.h

```diff
@@ -9,28 +9,20 @@
 #define COMMON_COND_HANDLE_H_
 
 /**
- * We declare a type cond_handle_t that depends on the system condition variable
- * primitives
+ * We define a type cond_handle_t, depending on the system condition variable
```

Comment:
How about changing wording from "depending on" to "mapping"? It would behave as this commit did.